src/utils/appError.ts

```diff
@@ -1,12 +1,12 @@
 class AppError extends Error {
```

Comment:
I didn't notice Errors before, but first of all - very nice to use custom errors, but here is a better example how to implement them with nodejs (mostly meant for http servers)

https://github.com/LiudasDEM/aws-lambda-middleware#readme
https://github.com/LiudasDEM/aws-lambda-middleware/tree/main/lib/errors


this is a library published into npmjs, take a look :)